tests/fgcmcalTestBase.py

```diff
@@ -187,34 +187,33 @@ def _testFgcmBuildStars(self, visits, nStar, nObs):
         starObs = butler.get('fgcmStarObservations')
         self.assertEqual(nObs, len(starObs))
 
-    def _testFgcmBuildStarsTable(self, visits, nStar, nObs):
+    def _testFgcmBuildStarsAndCompare(self, visits):
         """
-        Test running of FgcmBuildStarsTableTask
+        Test running of FgcmBuildStarsTask and compare to Table run
 
         Parameters
         ----------
         visits: `list`
            List of visits to calibrate
-        nStar: `int`
-           Number of stars expected
-        nObs: `int`
-           Number of observations of stars expected
 
         Raises
         ------
         Exceptions on test failures
         """
-        args = [self.testDir, '--output', os.path.join(self.testDir, 'rerun', 'sourceTable'),
+        args = [self.testDir, '--output', os.path.join(self.testDir, 'rerun', 'src'),
                 '--id', 'visit='+'^'.join([str(visit) for visit in visits]),
                 '--doraise']
         if len(self.configfiles) > 0:
             args.extend(['--configfile', *self.configfiles])
         args.extend(self.otherArgs)
 
-        result = fgcmcal.FgcmBuildStarsTableTask.parseAndRun(args=args, config=self.config)
+        result = fgcmcal.FgcmBuildStarsTask.parseAndRun(args=args, config=self.config)
         self._checkResult(result)
 
-        asdflkjljk
+        butlerSrc = dafPersist.Butler(os.path.join(self.testDir, 'rerun', 'src'))
+        butlerTable = dafPersist.Butler(os.path.join(self.testDir))
+
+        self._compareBuildStars(butlerSrc, butlerTable)
```

Comment:
Worth noting here that the output of these two may not be identical, even though they are run on the same data?